massgen/tests/test_cb_store_adversarial.py

```
@@ -0,0 +1,1062 @@
+        with pytest.raises(RuntimeError, match="expire failed"):
+            store.set_state("backend", _complete_state(state="open", failure_count=3))
+
+        # After partial write: HSET succeeded (data readable), EXPIRE failed (no TTL).
+        # State is defined -- either full data if HSET was atomic, or empty if HSET
+        # also failed. Both are acceptable; no crash and no corrupted type errors.
+        try:
+            state = store.get_state("backend")
+            # If data was written, it should be structurally valid.
+            assert isinstance(state["failure_count"], int)
+            assert state["state"] in ("closed", "open", "half_open")
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)